props/saber_fett263_buttons.h

```diff
@@ -4725,8 +4753,12 @@ SaberFett263Buttons() : PropBase() {}
           MenuChoice();
           return true;
         }
-        ToggleBattleModeMultiBlast();
-        SaberBase::DoBlast();
+        if (CheckInteractiveBlast()) {
+          SaberBase::DoEffectR(EFFECT_FIRE);
```

Comment:
I'm confused, shouldn't this be SaberBase::DoEffectR(EFFECT_INTERACTIVE_BLAST);
?

Reply:
Originally when I first built we were using EFFECT_FIRE (borrowed from blaster) but I've added INTERACTIVE_BLAST and updated